recipes/starshot_card_block_type/config/core.entity_form_display.block_content.card.default.yml

```diff
@@ -0,0 +1,58 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - block_content.type.card
+    - field.field.block_content.card.body
+    - field.field.block_content.card.field_card_link
+    - field.field.block_content.card.field_card_media
+  module:
+    - link
+    - media_library
+    - text
+id: block_content.card.default
+targetEntityType: block_content
+bundle: card
+mode: default
+content:
+  body:
+    type: text_textarea_with_summary
+    weight: 26
+    region: content
+    settings:
+      rows: 3
+      summary_rows: 3
+      placeholder: ''
+      show_summary: false
+    third_party_settings: {  }
+  field_card_link:
+    type: link_default
+    weight: 28
+    region: content
+    settings:
+      placeholder_url: ''
+      placeholder_title: ''
+    third_party_settings: {  }
+  field_card_media:
+    type: media_library_widget
+    weight: 27
+    region: content
+    settings:
+      media_types: {  }
+    third_party_settings: {  }
+  info:
+    type: string_textfield
+    weight: -5
```

Comment:
We tend to keep the info field for admin uses. We normally add a second title field and use a base field override to rename the 'info' field to 'Administrative label' because otherwise you can end up with hard-to-discern duplicates in the block library

Reply:
Plus 1 for administrative label

Reply:
Wait - is it an _administrative_ label, or the actual title of the card? I was thinking it was the latter.

Reply:
info is a base field that is used in admin pages ($entity->label()) but sometimes admins need more context to find a block in the library - so having two fields allows that

Reply:
Okay, I overrode the base field (for both block types) in 8afad325ef69d7407d4efc7585a9285a9d8b243f, along with adding a dev dependency that allows us to configure base field overrides in the UI.

Then I created a Title field for card blocks in 50b4f1a6cf919ea3408fa8f6c8ded0080d0a0899.

And then finally, I fixed an up-to-this-point oversight in 0be6973847666a98a148a5f9a1f9f2db4768cf99, making all fields translatable by default. This is out of scope for this PR, but eh...we're still in rapid prototyping, let's just fix the thing.